pycbc/dq.py

```diff
@@ -199,11 +200,12 @@ def query_flag(ifo, name, start_time, end_time,
                 for rseg in segs:
                     seg_start = rseg[0] + flag['start_pad']
                     seg_end = rseg[1] + flag['end_pad']
-                    flag_segments.append(segment(seg_start, seg_end))
+                    partial.append(segment(seg_start, seg_end))
 
-            # Apply start / end of the veto definer segment
-            send = segmentlist([segment([veto_def['start'], veto_def['end']])])
-            flag_segments = (flag_segments.coalesce() & send)
+                # Apply start / end of the veto definer segment
```

Comment:
Not sure what this is supposed to do & help is vague.  What are flag['start'] and flag['end']?

Reply:
Changed comment to "Limit to the veto definer stated valid region of this flag". It's just applying what the veto definer says are the boundaries that this flag should be used.